web3-services/gas-api/supported-networks.md

```diff
@@ -0,0 +1,89 @@
+---
+description: Supported networks.
+sidebar_position: 2
+---
+
+# Supported networks
+
+The Gas API supports multiple networks.
+
+Specify the chain ID in your request to interact with the relevant network.
+
+### Arbitrum
```

Comment:
These headings are at level 3 instead of level 2; is that intentional? So, the headings for the rendered doc are smaller?